WebDriverAgentLib/Utilities/FBRunLoopSpinner.m

```diff
@@ -71,7 +72,7 @@ - (BOOL)spinUntilTrue:(FBRunLoopSpinnerBlock)untilTrue error:(NSError **)error
 {
   NSDate *timeoutDate = [NSDate dateWithTimeIntervalSinceNow:self.timeout];
   while (!untilTrue()) {
-    [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:self.interval]];
```

Comment:
are there other places in the code where runUntilDate is used or these are the only ones?